app/components/user-page/course-progress-list.ts

```diff
@@ -0,0 +1,43 @@
+import Component from '@glimmer/component';
+import { groupBy } from 'codecrafters-frontend/utils/lodash-utils';
+import type UserModel from 'codecrafters-frontend/models/user';
+import type CourseParticipationModel from 'codecrafters-frontend/models/course-participation';
+
+type Signature = {
+  Element: HTMLDivElement;
+
+  Args: {
+    user: UserModel;
+  };
+};
+
+export default class CourseProgressListComponent extends Component<Signature> {
+  get courseParticipationGroups() {
+    const groupedCourseParticipations: CourseParticipationModel[][] = Object.values(groupBy(this.args.user.courseParticipations, 'course'));
```

Comment:
```suggestion
    const participationsGroupedByCourse: CourseParticipationModel[][] = Object.values(groupBy(this.args.user.courseParticipations, 'course'));
```

More consistent, it's easier to figure out the difference between this variable and the one below if the only difference in their name is the "sorted" prefix.